src/main.nqp

```diff
@@ -31,12 +31,12 @@ my $install-dir := $execname eq ''
     ?? $comp.config<prefix>
     !! nqp::substr($execname, 0, nqp::rindex($execname, $sep, nqp::rindex($execname, $sep) - 1));
 
-my $perl6-home := $comp.config<static_perl6_home>
-    // nqp::getenvhash<RAKU_HOME>
+my $rakudo-home := $comp.config<static_rakudo_home>
+    // nqp::getenvhash()<RAKUDO_HOME>
```

Comment:
This already done in the release branch and merged back into master. Also, for the start period, I prioritized `PERL6_HOME` over `RAKUDO_HOME`. In ~4-6 months we can prioritize `RAKUDO_HOME` and deprecate `PERL6_HOME` soon after or right away. But this is yet to be considered.